### PR TITLE
Enable sending comments from slack console

### PR DESCRIPTION
**Pull request by pawurb**
@aarek @jestemkarol please review

### CODE REVIEW

File: app/controllers/slack/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      response_type: "in_channel",
+      text: "Your channel is not assigned to any active debate."
+    }
+  end
+
+  private
+
+  def comment_maker_params
+    {
+      user_id: user&.id,
```
Comment (jestemkarol) [inline app/controllers/slack/comments_controller.rb:37]: some typo?

File: app/serializers/slack/comment_serializer.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class Slack::CommentSerializer < Struct.new(:comment)
```
Comment (aarek) [inline app/serializers/slack/comment_serializer.rb]: [Struct doc](http://ruby-doc.org/core-2.3.3/Struct.html#method-c-new) don't recommend extending Struct class. use `Struct.new(...) do ... end`

File: app/controllers/slack/comments_controller.rb (hunk 1 above)
Comment (pawurb) [reply app/controllers/slack/comments_controller.rb:37]: you mean & ? It is a ruby alternative to try

File: app/services/slack/user_maker.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    Slack::CommentMaker.call(params.merge(user_id: user.id))
+    user
+  rescue ApiNetworkError => e
+    Rails.logger.error e
+    false
+  end
+
+  private
+
+  def fetch_slack_api_data(user_slack_id)
```
Comment (aarek) [inline app/services/slack/user_maker.rb]: What do you think about extracting slack communication to another class?

**Review by aarek — APPROVED** [review]

**Pull request merged**